Code/Data_Files/Readme.md

```diff
@@ -0,0 +1,7 @@
+## Data files origin 
+
+- `Overlap_First_Interactors_NoDuplicates.xlsx` 
+    - script that use it: `Diversity_Histogram.py`
+    - source:
+        - what is described in this file? 
+        - how was created this file? 
```

Comment:
File was created from [Overlap_First_Interactors](https://github.com/leilaicruz/diauxic-shift-and-polarity-interactors/blob/master/Code/Data_Files/Overlap_First_Interactors.xlsx) by removing duplicate interactions